sw/device/tests/uart_baud_rate_test.c

```diff
@@ -24,16 +24,10 @@ static const uint32_t kBaseAddrs[4] = {
     TOP_EARLGREY_UART2_BASE_ADDR,
     TOP_EARLGREY_UART3_BASE_ADDR,
 };
-static uint32_t kBauds[7] = {
-    9600, 115200, 230400, 128000, 256000, 1000000, 1500000,
-};
+static uint32_t kBauds[5] = {9600, 115200, 230400, 128000, 256000};
```

Comment:
Could you make it `const` to justify the name pattern?
I think that we should test at least a sub set of the standard baud `4800, 9600, 19200, 38400, 57600, 115200, 230400, 460800,` Hyperdebug should be able to handle it.

Reply:
I've added most of these, except `460800` was rejected by the DIF so I've left that one out.